src/classes/TDTM_DMLgt200_TEST.cls

```diff
@@ -0,0 +1,465 @@
+                    LastName = 'TestContact# ' + n,
+                    Title = 'TestTitle'
+            ));
+        }
+        database.insert(contacts);
+        system.AssertEquals(insertCnt, [SELECT Count() FROM Account WHERE RecordType.DeveloperName = 'HH_Account'],
+                'There should be exactly ' + insertCnt.format() + ' Household Account records');
+
+        // Create Relationships between the Contacts
+        List<npe4__Relationship__c> rels =  new List<npe4__Relationship__c>();
+        for (Integer n=0; n<insertCnt; n++) {
+            Contact c1 = contacts[n];
+            Contact c2 = contacts[insertCnt - n - 1];
+            npe4__Relationship__c rel = new npe4__Relationship__c(
+                    npe4__Contact__c = c1.Id,
+                    npe4__RelatedContact__c = c2.Id,
+                    npe4__Type__c = 'Spouse',
+                    npe4__Description__c = 'InsertTest'
+            );
+            rels.add(rel);
+        }
+        database.insert(rels);
+        Test.stopTest();
+        system.AssertEquals(insertCnt*2, [SELECT Count() FROM npe4__Relationship__c
+        WHERE npe4__Contact__c IN :contacts AND npe4__Type__c = 'Spouse'],
+                'There should be exactly ' + (insertCnt*2).format() + ' Relationships');
+    }
+
+    /*********************************************************************************************************
+    * @description Test creating 200+ Opportunities with a default allocation through a Campaign
+    *   NOTE: THIS WILL FAIL IN AN ORG WITHOUT THE STATIC FLAG CHANGES
+    */
+    @isTest
+    static void test_opportunities_with_default_alloc() {
+        if (strTestOnly != '*' && strTestOnly != 'test_opportunities_with_default_alloc') return;
+
+        Contact c = UTIL_UnitTestData_TEST.getContact();
+        insert c;
+        c = [SELECT Id, AccountId FROM Contact WHERE Id = :c.Id LIMIT 1];
+
+        General_Accounting_Unit__c gau1 = new General_Accounting_Unit__c(
+                Name = 'TestGAU',
+                Active__c = True
+        );
+        General_Accounting_Unit__c gau2 = new General_Accounting_Unit__c(
+                Name = 'DefaultForCampaign',
+                Active__c = True
+        );
+        insert new List<General_Accounting_Unit__c>{gau1, gau2};
+
+        UTIL_CustomSettingsFacade.getAllocationsSettingsForTests(new Allocations_Settings__c(
+                Default_Allocations_Enabled__c = true, Default__c = gau1.Id));
+
+        Campaign camp = new Campaign(Name = 'UNITTEST', IsActive = True);
+        insert camp;
+
+        Allocation__c campAlloc = new Allocation__c(
+                Campaign__c = camp.Id,
+                General_Accounting_Unit__c = gau2.Id,
+                Percent__c = 100
+        );
+        insert campAlloc;
+
+        Test.startTest();
+
+        // Create 200+ Opportunity Records
+        List<Opportunity> opps = new List<Opportunity>();
+        for (Integer n=0; n<insertCnt; n++) {
```

Comment:
consider UTIL_UnitTestData_TEST.OppsForContactList() or one of the other helpers?